packages/server/__tests__/arpa_reporter/server/services/full-file-export.spec.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+
+const { expect } = require('chai');
+const sinon = require('sinon');
+
+const full_file_export = require('../../../../src/arpa_reporter/services/full-file-export');
+const aws = require('../../../../src/lib/gost-aws');
+
+const OLD_AUDIT_REPORT_BUCKET = process.env.AUDIT_REPORT_BUCKET;
+const OLD_ARPA_FULL_FILE_EXPORT_SQS_QUEUE_URL = process.env.ARPA_FULL_FILE_EXPORT_SQS_QUEUE_URL;
+
+describe('FullFileExport', () => {
+    const sandbox = sinon.createSandbox();
+    beforeEach(() => {
+        // Override environment variables for testing
+        process.env.AUDIT_REPORT_BUCKET = 'arpa-audit-reports';
+        process.env.ARPA_FULL_FILE_EXPORT_SQS_QUEUE_URL = 'http://sqs.us-west-2.localhost.localstack.cloud:4566/000000000000/full-file-export-queue';
```

Comment:
Given that calls to SQS are (should) be mocked, it might be better for this to be something that definitely shouldn't work, e.g.

```suggestion
        process.env.ARPA_FULL_FILE_EXPORT_SQS_QUEUE_URL = 'http://sqs.example.com/full-file-export-queue';
```
The idea being that this prevents any unit tests from getting a false positive due to a missing mock while LocalStack is running in a dev environment.

Reply:
Done 👍